pkg/irondb-backend.go

```diff
@@ -78,9 +78,10 @@ func (td *SampleDatasource) QueryData(ctx context.Context, req *backend.QueryDat
 		queryURL := req.PluginContext.DataSourceInstanceSettings.URL
 
 		td.circ, err = circ.New(&circ.Config{
-			TokenKey: key,
-			TokenApp: "circonus-irondb-datasource",
-			URL:      queryURL,
+			TokenKey:   key,
+			TokenApp:   "circonus-irondb-datasource",
+			URL:        queryURL,
+			MaxRetries: 0, // Do not retry 5XX error as doing so could lead to a flood of API requests.
```

Comment:
```suggestion
			MaxRetries: 0, // Do not retry 5XX error as Grafana is responsible for retry behavior
```

Reply:
I have updated the comment to make it better grammatically, and to replace the language you suggested.